src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,80 @@
+package main.java.duke;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> taskArr;
+    private int currTask;
```

Comment:
I suppose `currTask` is used to track the number of tasks & thus size of the `TaskList`? But in the method `size()`, `taskArr.size()` was called, so there wouldn't be a need for `currTask` then?
```suggestion
```